src/control/cmd/agent/main.go

```diff
@@ -40,19 +41,30 @@ import (
 	"github.com/daos-stack/daos/src/control/logging"
 )
 
+var daosVersion string
+
 const (
 	agentSockName        = "agent.sock"
 	daosAgentDrpcSockEnv = "DAOS_AGENT_DRPC_DIR"
 )
 
 type cliOptions struct {
-	AllowProxy bool   `long:"allow-proxy" description:"Allow proxy configuration via environment"`
-	Debug      bool   `short:"d" long:"debug" description:"Enable debug output"`
-	JSON       bool   `short:"j" long:"json" description:"Enable JSON output"`
-	ConfigPath string `short:"o" long:"config-path" description:"Path to agent configuration file" default:"etc/daos_agent.yml"`
-	Insecure   bool   `short:"i" long:"insecure" description:"have agent attempt to connect without certificates"`
-	RuntimeDir string `short:"s" long:"runtime_dir" description:"Path to agent communications socket"`
-	LogFile    string `short:"l" long:"logfile" description:"Full path and filename for daos agent log file"`
+	AllowProxy bool       `long:"allow-proxy" description:"Allow proxy configuration via environment"`
+	Debug      bool       `short:"d" long:"debug" description:"Enable debug output"`
+	JSON       bool       `short:"j" long:"json" description:"Enable JSON output"`
+	ConfigPath string     `short:"o" long:"config-path" description:"Path to agent configuration file" default:"etc/daos_agent.yml"`
+	Insecure   bool       `short:"i" long:"insecure" description:"have agent attempt to connect without certificates"`
+	RuntimeDir string     `short:"s" long:"runtime_dir" description:"Path to agent communications socket"`
+	LogFile    string     `short:"l" long:"logfile" description:"Full path and filename for daos agent log file"`
```

Comment:
I agree with David, it is strange that these lines appear as modified

Reply:
Adding the version subcommand to this struct caused a reformat due to the width.

Reply:
Oops, I have missed the formatting change... Thx and sorry Mike!

Reply:
Oops, sorry to have missed that!